config/lib/socket.io.js

```diff
@@ -85,7 +85,7 @@ module.exports = function (app, db) {
 
```

Comment:
The stuff above here talks about sessionSecret sessionId not sure if that's need/was causing my problem.